### ISSUE TITLE
App does not work any longer if GPS is deactivated 

### ISSUE THREAD

**Issue by bhintermayr** [corona-warn-app/cwa-wishlist#160]
Error occurs on different mobile devices.

Error message 'Standortdienste deaktiviert' . If I am turning on the GPS (which usually not do) the app works and states "Risikoermittlung aktiv".

This behaviour is new, first recognized today.

---
Internal Tracking ID: EXPOSUREAPP-2215

**Comment by MikeMcC399** [comment]
CWA Android 1.5 Data Privacy statement now says:

https://www.coronawarn.app/en/privacy/ > https://www.coronawarn.app/assets/documents/cwa-privacy-notice-en.pdf

In 8. What permissions does the app require? 
b. Android smartphones 

"If you have a smartphone running on Android version 10 or lower, location services need to be enabled for your smartphone to search for Bluetooth signals from other smartphones. Please note that no location data is collected in this process."

It's also available through
CWA > three-dot-menu > App Information > Data Privacy

By implication, it means location services do not need to be enabled on Android 11. 

I don't have an Android 11 physical device to try it out on, but on a Pixel 3a, Android 11 emulator with "Use location" disabled, Exposure Notifications did not show any warning.

(On Android 10, as expected, there was a notification
Exposure Notifications inactive
To use this feature, turn on location.)

So, in summary it seems to be working and documented as expected in Android 11. In Android 10 and below there are no changes and Location Services need to be enabled to allow Bluetooth Low Energy to work.

**Comment by daimpi** [comment]
@MikeMcC399 
> So, in summary it seems to be working and documented as expected in Android 11. In Android 10 and below there are no changes and Location Services need to be enabled to allow Bluetooth Low Energy to work.

Yes that seems to be correct except for a minor UI element which hasn't been adjusted for Android 11 yet but the devs [are already on it](https://github.com/corona-warn-app/cwa-app-android/issues/986#issuecomment-718856339) 🙂.

**Comment by MikeMcC399** [comment]
@daimpi 
Thanks for the pointer to https://github.com/corona-warn-app/cwa-app-android/issues/986#issuecomment-718856339!

I could not reproduce this problem with CWA Android 1.5.1.

The screenshot looks like it comes from an update, so I also tried installing 1.3.1 first, with "Use location" disabled. 1.3.1 shows the status LOCATION SERVICES DEACTIVATED. After updating to 1.5.1 the status changed to EXPOSURE LOGGING ACTIVE. I didn't see any message requesting location activation. So it looks like the issue has been fixed. Let me make a note in https://github.com/corona-warn-app/cwa-app-android/issues/986 about this.

**Comment by MikeMcC399** [comment]
I believe this issue has been resolved now. The app information is now correct.

The FAQ article https://www.coronawarn.app/en/faq/#android_location talks about it.

Is there something I missed or can this issue now be closed?

**Comment by heinezen** [comment]
@MikeMcC399 True, thank you. it has been obsolete for a while according to the internal ticket. I don't know how this ended up on the wishlist.

Closing because the issue has been resolved.

----
Corona-Warn-App Open Source Team
